README.md

```diff
@@ -1,3 +1,36 @@
 # TournaTrack
+    
 
-created by shubham
\ No newline at end of file
+  # Back-end
+[] NodeJs - setting up a project with package.json
```

Comment:
Pls Tick the checkboxes for which the topic are covered until now.

Reply:
Done